src/os_crypto/hmac/hmac.c

```diff
@@ -84,8 +84,8 @@ int OS_HMAC_SHA1_File(const char *key, const char *file_path, os_sha1 output, in
 
     key_length = strlen(key);
 
+    os_sha1 sha_key;
     if (key_length > HMAC_SHA1_BLOCKSIZE){
```

Comment:
When this condition is true, the parameter "key" returns an invalid pointer that points to a non existing local variable "sha_key". This could be a problem if the calling function tries to use it.

Reply:
done